doc/whats_new/v0.20.rst

```diff
@@ -44,6 +44,13 @@ Classifiers and regressors
   Naive Bayes classifier described in Rennie et al. (2003).
   By :user:`Michael A. Alcorn <airalcorn2>`.
 
+- :class:`linear_model.SGDClassifier`, :class:`linear_model.SGDRegressor`,
+  :class:`linear_model.PassiveAggressiveClassifier`,
+  :class:`linear_model.PassiveAggressiveRegressor` and
+  :class:`linear_model.Perceptron` now expose a ``early_stopping`` and
+  ``validation_fraction`` parameters, to stop optimization monitoring the
+  score on a validation set. :issue:`9043`  by `Tom Dupre la Tour`_.
```

Comment:
Add another entry for adaptive learning rate, or put it here. I'm not sure if each estimator needs to be listed. You can reference the user guide instead...?